src/js/components/Form/stories/UnsolicitedFeedback.js

```diff
@@ -0,0 +1,133 @@
+import React, { useContext, useState } from 'react';
+import styled from 'styled-components';
+import {
+  Box,
+  Button,
+  Form,
+  Layer,
+  FormField,
+  Text,
+  TextArea,
+  Header,
+  Heading,
+  ThumbsRating,
+  ResponsiveContext,
+} from 'grommet';
+
+// create a floating button for story example
+const PositionedFeedbackButton = styled(Button)`
+  position: fixed;
+  bottom: 0px;
+  border-radius: 6px;
+  right: 0px;
+  z-index: 10;
+`;
+
+// This example shows a way to perform validation across multiple fields.
+export const UnSolicitedFeedback = () => {
+  const size = useContext(ResponsiveContext);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  const closeFeedbackModal = () => {
+    setTimeout(() => {
+      setOpen(false);
+      setIsSuccessful(false);
+    }, 3000);
+  };
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <>
+      <PositionedFeedbackButton
+        onClick={onOpen}
+        margin={{ vertical: 'medium', horizontal: 'medium' }}
+        elevation="large"
+        color="brand"
+        label="Submit Feedback"
+        a11yTitle="This button launches a modal to give feedback."
+        primary
+        alignSelf="start"
+      />
+      {open && (
+        <Layer onEsc={onClose}>
+          <Box
+            fill="vertical"
+            overflow="auto"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+            pad="medium"
+            gap="medium"
+          >
+            <Header
+              direction="column"
+              align="start"
+              gap="xxsmall"
+              pad={{ horizontal: 'xxsmall' }}
+            >
+              <Heading level={4} size="small" margin="none">
+                Let us know how your expirence was!
+              </Heading>
+            </Header>
+            <Box
+              // Padding used to prevent focus from being cutoff
+              pad={{ horizontal: 'xxsmall' }}
+            >
+              <Form
+                method="post"
+                validate="submit"
+                kind="survey"
+                onSubmit={(value) => {
```

Comment:
After clicking Submit I am forced to wait until the Layer closes itself. I feel like you should still be able to close the Layer by hitting 'esc' or something so you are not forced to wait

Reply:
I can add `esc` thats just the pattern I was following from the DS example

Reply:
added `onClickOutside={onClose}`